Sprint-1/implement/max.js

```diff
@@ -1,4 +1,24 @@
 function findMax(elements) {
+
+// 〰️ Filter the array to include only numbers
+  const numericElements = elements.filter((el) => typeof el === "number");
+
+  // 〰️ Return -Infinity for an empty array (or no numeric values)
+  if (numericElements.length === 0) {
+    return -Infinity;
+  }
+
+  // 〰️ Use Math.max to find the largest number in the array
+  return Math.max(...numericElements);
```

Comment:
#### Sprint-1/implement/max.js
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```
---

Reply:
Hi,

I added the !isNaN(el) condition to the function. Now, it works as expected for all extreme cases mentioned, correctly handling -Infinity, NaN, and arrays with mixed values.

findMax([-Infinity]): The function correctly returns -Infinity because it is the only number in the array.
findMax([-Infinity, 0]): The function returns 0, as 0 is greater than -Infinity.
findMax([NaN]): The function correctly returns -Infinity, since NaN is excluded due to the && !isNaN(el) check, ensuring that NaN values are ignored.
findMax([0, NaN, 1]): The function returns 1, as NaN is ignored, and the largest number in the array is 1.

Thank you for your feedback!